examples/seismic/model.py

```diff
@@ -590,19 +616,24 @@ def critical_dt(self):
 
     @property
     def vp(self):
-        """:class:`numpy.ndarray` holding the model velocity in km/s.
-        .. note::
+        """`numpy.ndarray` holding the model velocity in km/s.
```

Comment:
nitpicking: by convention, all docstrings start on a new line, that is
instead of
```
"""`numpy.ndarray` holding ...
```
we should have
```
"""
`numpy.ndarray` holding ...
```
here and ...everywhere

This is really nitpicking though